config/locales/client.en.yml

```diff
@@ -3720,6 +3720,7 @@ en:
         group_members: "Group members"
         delete: "Delete"
         delete_confirm: "Delete this group?"
+        delete_with_messages_confirm: "Deleting this group will cause %{count} messages to be orphaned, group members will no longer have access to them.<br><br>Are you sure?"
```

Comment:
We probably want two messages here, for one and other. Otherwise it would say "1 messages" instead of "1 message"